admin/app/components/solidus_admin/ui/toast/component.html.erb

```diff
@@ -0,0 +1,22 @@
+<div
+  class="<%= [
+    stimulus_id,
```

Comment:
It won't hurt, but I'd like to understand if we need the `simulus_id` as class for some reason :thinking:

Reply:
That's a good point, probably not but it was coming from the template created with the component generator. @elia insights here?